index.js

```diff
@@ -175,6 +184,15 @@ exports.valid_hostname = function (next, connection, helo) {
     return next()
   }
 
+  // Check whether the HELO hostname is a valid ASCII string
+  if (!/[\x20-\x7E]+/.test(helo)) {
+    connection.results.add(this, { fail: 'valid_hostname(not_ascii)' })
+    if (this.cfg.reject.valid_hostname) {
```

Comment:
```suggestion
    this.cfg.reject.valid_hostname ? next(DENY, DSN.helo_not_ascii()) : next()
    return
```

This seems more tidy. What do you think?